<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSensore</class>
 <widget class="QDialog" name="AddSensore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>347</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 65, 74);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nuovo Sensore</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nome:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sensorName">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tipo di sensore:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sensorType">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 236, 237);
background-color: rgb(0, 65, 74);
border-radius: 5px;
padding: 0.3em;</string>
       </property>
       <property name="locale">
        <locale language="Italian" country="Italy"/>
       </property>
       <item>
        <property name="text">
         <string>temperatura</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>umidit√†</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fumo</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="maxVal">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Temperatura Massima:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sensorTempMax">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 236, 237);
background-color: rgb(0, 65, 74);
border-radius: 5px;
padding: 0.3em;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="textVar">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sensorVarValue">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 236, 237);
background-color: rgb(0, 65, 74);
border-radius: 5px;
padding: 0.3em;</string>
       </property>
       <property name="locale">
        <locale language="Italian" country="Italy"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutTemp">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="tempMinLabel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="locale">
        <locale language="Italian" country="Italy"/>
       </property>
       <property name="text">
        <string>Temperatura Minima:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="tempMinima">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 236, 237);
background-color: rgb(0, 65, 74);
border-radius: 5px;
padding: 0.3em;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="annullaButton">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 236, 237);
background-color: rgb(0, 65, 74);
border-radius: 5px;
padding: 0.3em;</string>
       </property>
       <property name="text">
        <string>Annulla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confermaButton">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 236, 237);
background-color: rgb(0, 65, 74);
border-radius: 5px;
padding: 0.2em;</string>
       </property>
       <property name="text">
        <string>Aggiungi</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
